style.css

```diff
@@ -643,10 +703,73 @@ footer::after {
   border-radius: 8px;
 }
 
-.social-media-footer {
-  list-style: none;
-  padding: 0;
-  display: flex;
-  justify-content: center;
-  width: 170px;
+@media (min-width: 768px) {
+  html,
+  body {
+    padding-bottom: 0;
+    margin-bottom: 0;
+  }
+
+  footer > a {
+    display: none;
+  }
+
+  footer::after {
+    content: none;
+  }
+
+  .footer-container {
+    margin-bottom: 0;
+    padding-bottom: 0;
+    background: url(img/illust1-desk-cntct.svg) no-repeat 0% 90%, url(img/illust2-desk-cntct.svg) no-repeat 30% 90%, url(img/illust3-desk-cntct.svg) no-repeat 100% 90%;
+  }
+
+  .footer-form {
+    display: flex;
+    flex-direction: row;
+    text-align: left;
+  }
+
+  .input-name {
+    display: none;
+  }
+
+  .input-full-name {
+    display: flex;
+    justify-content: space-between;
+    padding: 0;
+  }
+
+  .footer-container > .social-media {
+    margin-top: 18em;
+  }
+
+  .input-full-name,
+  .input-email,
+  textarea {
+    min-width: 327px;
+    width: calc(95vw - 684px);
+  }
+
+  textarea {
+    height: 114px;
```

Comment:
- [ ] The color of the border of the input fields should be green(ish) when inputting values:

<img width="434" alt="Zrzut ekranu 2022-03-24 o 21 00 40" src="https://user-images.githubusercontent.com/87897753/160000542-f8298c13-3688-4b29-8853-24718e99081d.png">